problemsets-mat223/modules/module3.tex

```diff
@@ -105,7 +105,28 @@
 \begin{example}
 	Let $\vec u=\mat{-1\\2}$ and $\vec v=\mat{1\\-2}$. Find $\Span\Set{\vec u,\vec v}$.
 
-	XXX Finish
+	By the definition of span,
+	\[
+	\Span\Set{\vec u,\vec v}=
+	\Set{\vec x\given \vec x=\alpha\vec u+\beta\vec v\text{ for some scalars }\alpha,\beta}.
+	\]
+	Solving for when $\mat{x\\y} = \alpha\mat{-1\\2}+\beta\mat{1\\-2}$ is consistent will
+	let us find the vectors that can be expressed as a linear combination of 
+	$\vec u$ and $\vec v$.
+	
+	We can solve the equation by considering the equations arising from the first and second coordinates.
+	\begin{alignat*}{3}
```

Comment:
Search through the soucecode for `\systeme`. It will automatically typeset a system of equations for you :-)